salt/kubelet/status.sls

```diff
@@ -0,0 +1,5 @@
+{% set localhost = salt['grains.get']('localhost', '') %}
```

Comment:
The kubelet register nodes under `grains['nodename']`, not `grains['localhost']` (and they don't have to match), so you could be trying to check the status of an inexistent node...